central/networkgraph/aggregator/aggregator.go

```diff
@@ -46,3 +46,7 @@ func NewDefaultToCustomExtSrcConnAggregator(networkTree tree.ReadOnlyNetworkTree
 func NewDuplicateNameExtSrcConnAggregator() NetworkConnsAggregator {
 	return &aggregateExternalConnByNameImpl{}
 }
+
+func NewAllAggregator() NetworkConnsAggregator {
```

Comment:
This name is a little vague - might be better as `NewLatestTimestampAggregator` to demonstrate which information it's using to aggregate. (same for the `aggregateAllImpl` struct)

Reply:
Done